emc2/simulator/subcolumn.py

```diff
@@ -322,7 +322,7 @@ def set_precip_sub_col_frac(model, is_conv, N_columns=None, use_rad_logic=True,
         print(f"num_subcolumns == 1 (subcolumn generator turned off); setting subcolumns frac "
               f"fields to 1 for {precip_type} precip based on q > 0. kg/kg")
         for hyd_type in precip_types:
-            if is_conv:
```

Comment:
I'd recommend retaining the is_conv option (more intuitive) but switching between l. 326 and 328.